pipeline/notifications/rating_reminder.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from pipeline.notifications.notifier import Notifier
+from pipeline.types import NotificationInfo, NotificationInput, NotificationType
+
+
+class RatingReminderNotifier(Notifier):
+    def __init__(self):
+        pass
+
+    def get_notifications(
+        self, inp: NotificationInput
+    ) -> List[NotificationInfo]:
+        remind_users = []
+        min_chat: int = 3
+        numMessages = 0
+        listUsers = inp.chatdata.participants
+        chatMessages = inp.chatdata.messages
+        dateTime = inp.run_datetime
+        for messagesSent in chatMessages:
+            if messagesSent.from_user in listUsers:
+                numMessages += 1
+        if dateTime.weekday() == 5:
+            if numMessages >= min_chat:  # should be more than 3 msg bubbles
```

Comment:
I suggest we have this as one if statement instead of a nested-if statement. So, basically, **if dateTime.weekday() == 5 && numMessages >= min_chat**. Doing so also removes the need for the additional return statement on lines 33-36, where they share an identical return value anyway.